homeassistant/components/rfxtrx/__init__.py

```diff
@@ -115,14 +121,24 @@ def handle_receive(event):
         for subscriber in RECEIVED_EVT_SUBSCRIBERS:
             subscriber(event)
 
-    device = config[DOMAIN][ATTR_DEVICE]
+    device = config[DOMAIN].get(ATTR_DEVICE, None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
Fixed